src/components/HorizontalTabList/index.js

```diff
@@ -1,40 +1,23 @@
 import React, {Component} from 'react';
 import './styles.css';
 
-class HorizontalTabList extends Component {
-
-  getListItems() {
-    const list = this.props.list;
-    const listItems = [];
-
-    for (let i = 0; i < list.length; i++) {
-      const newItem = this.generateListItem(list[i].symbol, i);
-      listItems.push(newItem);
-    }
-
-    return listItems;
-  }
-
-  generateListItem(title, counter) {
-    if (title === this.props.selectedItem.symbol) {
-      return <li className="list-inline-item active" key={counter}>{title}</li>;
-    } else {
-      return <li className="list-inline-item" key={counter} onClick={() => this.changeSelectedItem(title)}>{title}</li>;
-    }
-  }
-
-  changeSelectedItem(title) {
-    this.props.changeSelected(title);
-  }
-
-  render() {
-    return (
-      <ul className="list-inline tablist">
-        {this.getListItems()}
-      </ul>
-    )
-  }
-
-}
-
-export default HorizontalTabList;
+export default class HorizontalTabList extends Component {
+  getListItems = () => this.props.list.map((item, i) => this.generateListItem(item.symbol, i));
```

Comment:
This method is too long (in chars).
```javascript
getListItems = () =>
  this.props.list.map((item, i) => this.generateListItem(item.symbol, i));
```

Reply:
If you think that method is long, you should have taken a look at the getButtons() method in OptionPanel 💯.